toolbox/db/db_get.m

```diff
@@ -44,14 +44,90 @@
     end
 end
 
+st = dbstack;
```

Comment:
Leftover debugging variable?

Reply:
Solved in 656d5cc318907a28da0d0c5823e1989782eb4ace